app/views/shared/check_answers/_assets.html.erb

```diff
@@ -13,11 +13,15 @@
 
 <section class="print-no-break">
   <%= render "shared/check_answers/vehicles", read_only:, individual: %>
+</section>
 
+<section class="print-no-break">
   <% if showing_bank_accounts %>
     <h2 class="govuk-heading-m"><%= t(".assets.bank_accounts") %></h2>
   <% end %>
+</section>
 
+<section class="print-no-break">
```

Comment:
Same as other comments; this goes around several headings and summary cards and should just go round one of each